plugin/action/join_go_panic/join_go_panic.go

```diff
@@ -0,0 +1,96 @@
+package join_go_panic
```

Comment:
You decided do separate plugins for go, java etc? It can be one plugin with different modes. But one plugin won't be simple

Reply:
Yes, I think  it's a good idea to create one plugin for all such cases, where another languages patterns could be added as new options. So I propose an implementation of [join_template plugin](https://github.com/ozontech/file.d/pull/152/files#diff-d49806fa9cd1fad1ddc6ed32bb2e74747f59eaf9428a64c01eb65f35517a21cc).